### ISSUE TITLE
Begin making the Tooltip Menu keyboard accessible

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons#10980]
STR:
- Go an add-on detail page
- Click on Read all reviews to see the list of reviews
- When the page loads, use only the tab key to navigate the page
- When you tab to the *flag* link press the space bar or enter key to flag a review
- Press tab to navigate the menu (this does not work)

The `TooltipMenu` component added in https://github.com/mozilla/addons/issues/10965 is not fully keyboard accessible. In the interest of preserving forward momentum, let's try and tackle this in a separate issue.

A helpful guide: http://heydonworks.com/practical_aria_examples/#progressive-collapsibles
